user_profile.py

```diff
@@ -56,12 +56,181 @@
 # Create the SQLAlchemy database object
 db.init_app(user_profile_app)
 
+# global resources fields to serialize the response object
+userInformation_resource_fields = {
+    'first_name' : fields.String,
+    'middle_name' : fields.String,
+    'last_name' : fields.String,
+    'age' : fields.Integer,
+    'date_of_birth' : fields.DateTime(dt_format='iso8601'),
+    'address_line_1' : fields.String,
+    'address_line_2' : fields.String,
+    'city' : fields.String,
+    'province' : fields.String,
+    'country' : fields.String,
+    'postal_code' : fields.String,
+    'gender' : fields.String,
+    'religion' : fields.String,
+    'profile_image' : fields.String,
+    'user_bio' : fields.String,
+    'interests' : fields.String,
+    'education_institutions' : fields.String,
+    'education_majors' : fields.String,
+    'education_degrees' : fields.String,
+    'graduation_date' : fields.DateTime(dt_format='iso8601'),
+    'identification_option' : fields.String,
+    'identification_material' : fields.String,
+    'user_id' : fields.String
+}
+
+
+
+
 
 # Querying and inserting into user profile database Flask_Restful API
 class UserInformationResource(Resource):
```

Comment:
If you're wondering, this class name follows the pascal case convention correctly.

So in Python, class name must be pascal case: `LikeThisExampleClass`, and all variables should be snake case:

- `this_variable_for_example`
- `_private_variable`
- `__private_method`